platform/osx/detect.py

```diff
@@ -76,6 +76,7 @@ def configure(env):
     elif env["target"] == "debug":
         env.Prepend(CCFLAGS=["-g3"])
         env.Prepend(CPPDEFINES=["DEBUG_ENABLED"])
+        env.Prepend(LINKFLAGS=["-Xlinker", "-no_deduplicate"])
```

Comment:
Doesn't it result in bigger, and possibly worse optimized binaries?

Reply:
For debug this doesn't matter IMHO, it sets these by default in xcode too normally for debug:
- Bin size: 198 MB with this setting on
- Bin size: 190 MB with this setting off

Before:
scons -j16  58.31s user 3.16s system 111% cpu 55.303 total

After:
scons -j16  20.81s user 3.21s system 141% cpu 17.007 total